app/views/setup/welcome.html.slim

```diff
@@ -20,23 +21,43 @@ h1 Initial CaaSP Configuration
               i class='glyphicon glyphicon-info-sign'
 
   .panel.panel-default
-    .panel-heading Container engine proxy settings
-    .panel-body
-      p Proxy server the container engine will use to download images.
-      p Leave them empty if no proxy server is in place inside of your organization.
-      .form-group
-        = f.label :http_proxy, "HTTP proxy"
-        = f.url_field :http_proxy, value: Pillar.value(pillar: :http_proxy), class: "form-control"
-      .form-group
-        = f.label :https_proxy, "HTTPS proxy"
-        = f.url_field :https_proxy, value: Pillar.value(pillar: :https_proxy), class: "form-control"
+    .panel-heading.clearfix
+      h3.panel-title Proxy settings
       .form-group
-        = f.label :no, "No-proxy"
-        .input-group
-          = f.text_field :no_proxy, value: Pillar.value(pillar: :no_proxy), class: "form-control"
-          span class="input-group-addon"
-            a data-toggle="tooltip" data-placement="left" title="Comma separated list of sites or domains that should not be accessed via the proxy server."
-              i class='glyphicon glyphicon-info-sign'
+        .input-group.btn-group-toggle.pull-right data-toggle="buttons"
```

Comment:
Just realized that you are using [input group](http://getbootstrap.com/components/#input-groups) instead of [button group](http://getbootstrap.com/components/#btn-groups). Apparently there's no difference but not sure if `.btn-group-sm` will work as expect if input group.